cg/apps/deliverables_metrics_parser/models/pipeline_metrics_deliverables.py

```diff
@@ -0,0 +1,20 @@
+"""Module for the pipeline metrics deliverables models."""
+from pydantic import BaseModel, Field
+
+
+class MIPDNAMetricsDeliverables(BaseModel):
```

Comment:
It'd be good to merge this model with `MetricsDeliverables`, including layers for common and pipeline-specific metrics, along with a Pydantic model for QC checks. Unsure if both implementations are necessary. 

Then we'll just have a simple static model for all metrics and use submodels to characterise them. But, as you mentioned in the code, I suppose we'll need to decide on deliverable formatting first (?).